src/main/java/friday/task/Event.java

```diff
@@ -0,0 +1,76 @@
+package friday.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an event task in the Friday application.
+ * Subclass of the Task class.
+ */
+public class Event extends Task {
+    protected LocalDateTime from;
+    protected LocalDateTime to;
+
```

Comment:
Should these fields be protected or private? Maybe setting them to private for their visibility might be better.

Reply:
okay! will improve!